src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -46,7 +46,7 @@ def get_simulant_id_maps(
         obs_results=obs_data,
         output_columns=output_cols_superset,
     )
-    maps.update(get_ssn_map(column_name, formatted_obs_data, randomness))
+    maps.update(get_ssn_map(column_name, formatted_obs_data, artifact))
```

Comment:
I am not sure I understand how this is working without a randomness stream?

Reply:
Generate SSNs never gets called it was just updated for the artifact?

Reply:
Yeah so the `get_ssn_map` happens without randomness stream b/c it load the already-randomly-shuffled SSN options and just picks off the top (no sampling needed b/c they're already shuffled).

The only time `generate_ssns` is called is when a simulant makes up a SSN on the spot, and that happens after the maps are already generated as part of the `for observer in FINAL_OBSERVERS:` loop